starter-core/src/main/java/io/micronaut/starter/feature/test/MicronautTestNettyLeak.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.test;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.util.StringUtils;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.build.dependencies.MicronautDependencyUtils;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.DefaultFeature;
+import io.micronaut.starter.feature.Feature;
+import io.micronaut.starter.options.Options;
+import io.micronaut.starter.options.TestFramework;
+import jakarta.inject.Singleton;
+
+import java.util.Set;
+
+@Requires(property = "micronaut.starter.feature.micronaut-test-netty-leak.enabled", value = StringUtils.TRUE, defaultValue = StringUtils.TRUE)
```

Comment:
i dont think this is useful enough for most people to become a default

Reply:
we add this `@Requires` for every feature.  It enables users to disable the feature.